cmd/format-xl.go

```diff
@@ -746,7 +746,7 @@ func initFormatXL(ctx context.Context, storageDisks []StorageAPI, setCount, driv
 	}
 
 	// Initialize meta volume, if volume already exists ignores it.
-	if err := initFormatXLMetaVolume(storageDisks, formats); err != nil {
```

Comment:
We shouldn't write .minio.sys on disks which don't have formats written successfully @vadmeste

Reply:
did you look at my comment? @vadmeste